lesson-2/script.js

```diff
@@ -0,0 +1,11 @@
+function calc() {
+    const firstNumber = +prompt('Enter first number');
+    const secondNumber = +prompt('Enter second numer');
+    alert(
+        `${firstNumber} + ${secondNumber} = ${firstNumber + secondNumber}
+${firstNumber} - ${secondNumber} = ${firstNumber - secondNumber}
+${firstNumber} * ${secondNumber} = ${firstNumber * secondNumber}
+${firstNumber} / ${secondNumber} = ${firstNumber / secondNumber}`);
+}
+
+calc()
```

Comment:
I don't like it when the indentation disappears when transferring lines. That's why I recommend this
```suggestion

     alert(
        `${firstNumber} + ${secondNumber} = ${firstNumber + secondNumber}` + '\n' +
        `${firstNumber} - ${secondNumber} = ${firstNumber - secondNumber}` + '\n' +
        `${firstNumber} * ${secondNumber} = ${firstNumber * secondNumber}` + '\n' +
        `${firstNumber} / ${secondNumber} = ${firstNumber / secondNumber}`
    );

calc();

```